jsonrpc/eth_endpoint.go

```diff
@@ -155,6 +155,41 @@ func (e *Eth) GetBlockByHash(hash types.Hash, fullTx bool) (interface{}, error)
 	return toBlock(block, fullTx), nil
 }
 
+// GetHeaderByNumber returns the requested canonical block header.
+// * When blockNr is -1 the chain head is returned.
+// * When blockNr is -2 the pending chain head is returned.
```

Comment:
Remove this part of comment, because it is not correct (for `PendingBlockNumber = -3` and `LatestBlockNumber = -2` we are returning the latest block number)
```suggestion
```

If you want, you can add such comment on the `GetNumbericBlockNumber` function.